app/controllers/cards_controller.rb

```diff
@@ -3,13 +3,11 @@ class CardsController < ApplicationController
   respond_to :html
 
   def index
-    @cards = current_user.cards.all
-    respond_with @cards
+    @cards = current_user.cards.all.order('review_date')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.